src/components/GPSButton/GPSButton.tsx

```diff
@@ -16,6 +17,7 @@ const StyledButton = styled(Button)`
   box-shadow: 0 0 0 2px rgba(0, 0, 0, 0.1);
   width: ${themeSpacing(36)};
   height: ${themeSpacing(11)};
+  color: ${themeColor('tint', 'level7')};
```

Comment:
Staan deze kleuren in een overzicht? Een soort gemeentelijke stylesheet?

Reply:
https://amsterdam.github.io/amsterdam-styled-components/?path=/story/introduction-welcome--page

en dan bij themecolor